vignettes/rpollution.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "rpollution"
+author: Christopher Alexander, Daniel King, Mel Liow
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{rpollution}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(rpollution)
+
+```
+
+```{r setup-hide, echo=FALSE}
+NOT_CRAN <- identical(tolower(Sys.getenv("NOT_CRAN")), "true")
+knitr::opts_chunk$set(purl = NOT_CRAN)
+
+my_secrets <- function() {
+  path <- "../config.json"
+  if (!file.exists(path)) {
+    stop("Can't find secret file: '", path, "'")
+  }
+  
+  jsonlite::read_json(path)
+}
+
+api_key <- my_secrets()$api_key
+
+```
+
+## Retrieve historic pollution data with specified date range and location:
+
+```{r historic-example}
+df <- get_pollution_history(1606488670, 1606747870, 49.28, 123.12, api_key)
+```
+
+```{r historic-example-result}
+head(df)
+```
+
+## Generate an interactive map containing current pollution data by location:
+
+```{r current-example}
+pollution_map_plot <- get_air_pollution(49.28, 123.12, api_key)
```

Comment:
There is an optional fig_title argument which might be worth showing off in the vignette

Reply:
Done!

Reply:
Oh sorry I was referring to the fig_title argument in my function lol

Reply:
Oh gotcha, okay will change it